plutus-benchmark/marlowe/test/Spec.hs

```diff
@@ -36,10 +36,10 @@ runTestGhc path =
 main :: IO ()
 main = do
   -- Read the semantics benchmark files.
-  semanticsMBench <- either error id <$> semanticsBenchmarks
+  semanticsMBench <- semanticsBenchmarks
```

Comment:
What happened here?

Reply:
I've noticed that in all places where the `semanticsBenchmarks` is used it is always accompanied by `either error id` error handling.  

I have changed the type of `semanticsBenchmarks` to not be `IO [Benchmark]` instead of `IO (Either String [Benchmark])` and it allowed to remove `either error id` everywhere it was used.